corehq/apps/hqwebapp/static/hqwebapp/js/mobile_experience_warning.js

```diff
@@ -0,0 +1,40 @@
+hqDefine('hqwebapp/js/mobile_experience_warning', function() {
+    if (window.innerWidth > 800 && window.innerHeight > 600) {
```

Comment:
Right now this check is in three places - any way to reduce that?

Reply:
Not really. I removed it from one of those places, since it wasn't even being used there. I like having the check happen in each place we can, though, since a user might have already migrated to desktop.